common/api/ecschema-metadata.api.md

```diff
@@ -420,7 +420,11 @@ export const Diagnostics: {
             readonly diagnosticType: DiagnosticType;
             ecDefinition: Schema;
             messageArgs?: [string, string] | undefined;
-            category: DiagnosticCategory;
+            category: DiagnosticCategory; /**

```

Comment:
I think you need to revert the changes to ecschema-metadata.api.md. extract-api behaves funkily with this package for some reason.

Reply:
Thanks. I actually saw it and didn't initially commit it as it didn't appear to have any affect other than comments and it seemed suspicious as to why it hadn't already been committed with them.

However, the build is complaining about this file, so I tried including it. I'll push a revert now.